docs/getting-started/switch-to-composer.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Switch to Composer"
+description: "How to switch your plugin based Timber theme over to the composer based version."
+menu:
+  main:
+    parent: "getting-started"
+---
+
+With the upcoming release of Timber 2, we will stop releasing Timber as a plugin. We advice everyone to switch to the composer based install as soon as possible.
+
+## Recommended upgrade path
+
+1. Get a local development version of your website up and running.
+2. Check if you are running the latest version of the Timber plugin, if not, update it and check if your website still runs as expected
+3. Disable the Timber plugin
+4. Install the latest 1.x version of Timber via composer
+5. Load the composer autoloader and initialize Timber 
+6. Check if your website still runs as expected
+7. Deploy your changes to your live website
+
+### 1. Get a local development version of your website up and running
+We highly recommend doing these steps on a local development version of your website. If you don't have one yet, you can use [Local by Flywheel](https://localbyflywheel.com/) to get one up and running in a few minutes.
+
+#### 1.1 How to install Composer
+If you don't have Composer installed yet, you can follow the [official installation guide](https://getcomposer.org/doc/00-intro.md).
+
+
+### 2. Check if you are running the latest version of the Timber plugin
+We want to make sure that you are running the latest version of the Timber plugin before we start the upgrade process. This way we can be sure that any issues that might occur are not caused by an outdated version of the plugin.
+
+### 3. Disable the Timber plugin
+Once you are sure that you are running the latest version of the Timber plugin, you can disable it. This will make sure that the plugin is not loaded anymore and does not interfere with the composer based version of Timber.
+
+### 4. Install the latest 1.x version of Timber via composer
+Now that the plugin is disabled, you can install the latest 1.x version of Timber via composer. You can do this by navigating to your site's active theme folder inside your terminal and then running the following commands:
+
+```shell
+composer init
```

Comment:
This init call and answering the questions is not needed at all. You just need to issue the "require" line to fetch the right files.